src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -447,18 +447,107 @@ TEST_F(SetGlobalPropertiesRequestTest, Run_VRWithMenuAndKeyboard_SUCCESS) {
   command->Run();
 }
 
+TEST_F(SetGlobalPropertiesRequestTest,
+       Run_VRWithMenuAndKeyboard_Menu_WARNINGS) {
```

Comment:
Test failures for these two new tests:
```
unknown file: Failure
C++ exception with description "
    The mock function has no default action set, and its return type has no default value set." thrown in the test body.
[  FAILED  ] SetGlobalPropertiesRequestTest.Run_VRWithMenuAndKeyboard_Menu_WARNINGS (2 ms)
[ RUN      ] SetGlobalPropertiesRequestTest.Run_VRWithMenuAndKeyboard_VR_WARNINGS
unknown file: Failure
C++ exception with description "
    The mock function has no default action set, and its return type has no default value set." thrown in the test body.
[  FAILED  ] SetGlobalPropertiesRequestTest.Run_VRWithMenuAndKeyboard_VR_WARNINGS (2 ms)
```

Seems an unexpected mock function is being called which has no defined behavior, causing a crash.

Reply:
@jacobkeeler yes, there were a few missing expectations in the unit tests. And NULL pointer dereference in the one of them, which is causing a core crash. Fixed in https://github.com/smartdevicelink/sdl_core/pull/2276/commits/51e40bf42f03ed368ac51123b77444313a2fe84d